src/app/static/src/tests/modelCalibrate/actions.test.ts

```diff
@@ -240,7 +240,9 @@ describe("ModelCalibrate actions", () => {
                         indicator_id: "test indicator",
                         x_axis_id: "test_x",
                         disaggregate_by_id: "test_dis",
-                        selected_filter_options: {"test_name": {}}
+                        selected_filter_options: {
+                            type: [{id: "test", label: "test"}]
```

Comment:
This data is actually representative of the format that is will be received when running the full app

Reply:
If you're just testing that some value gets shuffled around correctly by actions and mutations when it doesn't require its internal structure, it's often handy to just use terse dummy values rather than a full test object of the correct type. But this looks pretty terse anyway so fair enough.